lib/galaxy/tools/parameters/basic.py

```diff
@@ -151,7 +151,7 @@ def value_to_display_text( self, value, app ):
         Convert a value to a text representation suitable for displaying to
         the user
         """
-        if value:
+        if value is not None:
             return unicodify( value )
         return "Not available."
```

Comment:
Before this PR, integers with value `0` say `Not available.`, floats of value `0.0` say `Not available.`, booleans with a value of `False` say `Not available.`, properly set text inputs with a value of `''` say `Not available.`. 

After this PR, everything displays their actual Correct value, including the empty string.